Hunting Queries/SigninLogs/UserAccounts-UnusualLogonTimes.yaml

```diff
@@ -50,6 +56,35 @@ query: |
     | summarize count(), FirstSuccessfulSignin = min(TimeGenerated), LastSuccessfulSignin = max(TimeGenerated), make_set(IPAddress), make_set(ClientAppUsed), make_set(UserAgent), make_set(AppDisplayName) by HourOfLogin, Location, DayofWeek, UserPrincipalName
     ) on Location, DayofWeek , HourOfLogin
     | extend timestamp = LastSuccessfulSignin, AccountCustomEntity = UserPrincipalName
+  | join kind=leftouter (
+      IdentityInfo
+      | summarize LatestReportTime = arg_max(TimeGenerated, *) by AccountUPN
+      | extend BlastRadiusInt = iif(BlastRadius == "High", 1, 0)
+      | project AccountUPN, Tags, JobTitle, GroupMembership, AssignedRoles, UserType, IsAccountEnabled, BlastRadiusInt
+      | summarize
+          Tags = make_set(Tags),
+          GroupMembership = make_set(GroupMembership),
+          AssignedRoles = make_set(AssignedRoles),
+          BlastRadiusInt = sum(BlastRadiusInt),
+          UserType = make_set(UserType),
+          UserAccountControl = make_set(UserType)
+      by AccountUPN
+      | extend UserPrincipalName=tolower(AccountUPN)
+  ) on UserPrincipalName

```

Comment:
Sometime usernames in different logs can have different cases that can affect joins on them. Do SigninLogs and IdentifyInfo always have the same case? If not we should normalize the cases before joining.

Reply:
Added a tolower() for SigninLogs in cases where it is not clear since we are already doing that for IdentityInfo.